### ISSUE TITLE
Add a way to upload files from a git commit

### ISSUE THREAD

**Issue by parnanzio** [Natizyskunk/vscode-sftp#241]
Hello there! Is there a way to upload the files from a git commit (or a subset) with a single command?

A command like "Upload all files changed or created since the last commit to your Git", but for files already committed.

Or maybe directly from the Git Graph (don't know if it possible) or the git commit panel.

**Comment by Natizyskunk** [comment]
@parnanzio,
Nope, there is no such command for now. Maybe git support will be added later.
